.github/workflows/crawl.yml

```diff
@@ -0,0 +1,44 @@
+on:
+  schedule:
+    - cron: '30 */6 * * *'
+  push:
+    branches:
+    - master
+name: crawl
+jobs:
+  crawl:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: checkout reffy
+      uses: actions/checkout@master
+    - name: setup node
+      uses: actions/setup-node@master
+      with:
+        node-version: '10.x'
+    - name: install pandoc
+      run: sudo apt-get install pandoc
+    - name: setup reffy
+      run: |
+        echo "${{ secrets.CONFIG_JSON }}" | base64 --decode > config.json
+        npm ci
+    - name: reffy crawl
+      run: npm run all
+    - name: checkout reffy-reports
+      uses: actions/checkout@master
+      with:
+        repository: tidoust/reffy-reports
+        ref: master
+        path: reffy-reports
+    - name: copy reports
+      run: rsync -av --exclude=README.md reports/ ../reffy-reports/
+    - name: push reffy-reports
+      working-directory: ../reffy-reports
+      run: |
+        git config user.email auto@foolip.org
+        git config user.name autofoolip
+        git remote set-url --push origin https://autofoolip:${{ secrets.REFFY_REPORTS_TOKEN }}@github.com/foolip/reffy-reports.git
```

Comment:
An alternative to this would be to put the cron job in reffy-reports instead, and have that job clone/install reffy. That would avoid the need for a separate token. @tidoust do you have a preference for where the updating infra lives?

Reply:
Tied in with that is this question: should we do runs with the master branch of reffy, or `npm install reffy` to use the most recent published version?

Reply:
I think I'd prefer to put the logic in reffy-reports and leave the reffy repo as code-only, with reffy-reports containing the logic to generate the reports, the reports themselves, and the logic to update WPT.

Given that I haven't been consistent about updating the published version of Reffy on npm and that I don't see any good reason to incorporate changes in the master branch that we wouldn't want to see reflected in the next crawl, it may be better to rely on the master branch. I don't feel strongly either way.

Reply:
Alright, putting it in reffy-reports would avoid the need for `REFFY_REPORTS_TOKEN`, I can move to there and also figure out how to trigger the wpt sync on each update. (With GitHub Actions, a push done by one action can't trigger another action, so they'll need to happen together.)